Configurations.md

```diff
@@ -3096,11 +3096,11 @@ Note that no wrapping will happen if:
 1. The comment is the start of a markdown header doc comment
 2. An URL was found in the comment
 
-- **Default value**: `false`
-- **Possible values**: `true`, `false`
+- **Default value**: `"off"`
+- **Possible values**: `"doc"`, `"normal"`, `"all"` (alias `true`), `"off"` (alias `false`)
 - **Stable**: No (tracking issue: [#3347](https://github.com/rust-lang/rustfmt/issues/3347))
 
-#### `false` (default):
+#### `"off"` (default):
```

Comment:
Same comment as above about capitalizing the variants. Also, can you add a one or two sentence explanation mentioning that you can still set `false` for `Off`, and `true` for `All` under each respective section.

We should also add examples for `Doc` and `Normal` to highlight the formatting differences among all variants.